pykg2tbl/__main__.py

```diff
@@ -289,11 +282,9 @@ def main(sysargs=None):
     query = template_service.build_sparql_query(
         name=args.template_name, variables=params
     )
-    print("Making KGSource")
-    source = makesource(args)
     print("performing query")
     log.debug("making exec service")
-    executive_service = KG2TblService(source)
+    executive_service = KGSource.build(*args.source)
```

Comment:
please rename `executive_service` to simply `source`

so basically (with the updated makesource mentioned earlier) have
```python
    log.debug("building source")
    source = makesource(args)
    log.debug("performing query")
    result = source.exec(query)
    # then handle the os.path and delimitor stuff to serialize that output
```